application/ImageTabView.cpp

```diff
@@ -16,27 +16,26 @@
 
 #include "Tools.h"
 
-ImageTab::ImageTab(BView* tabView)
+ImageTab::ImageTab(const char* img_label, BView* tabView)
     :
     BTab(tabView)
 {
     fBitmap = nullptr;
+    fImageLabel = img_label;
```

Comment:
This seems quite fragile. As long as you are using string literals ("") it works, but std::string::c_str() passed as img_label will lead to crashes.
Please use BString for fImageLabel and change the variable name to fImageName, since it better describes the purpose.

Reply:
That's true. Thanks for pointing it out for me.